source/Ship.cpp

```diff
@@ -1204,6 +1204,8 @@ vector<string> Ship::FlightCheck() const
 		checks.emplace_back("no thruster!");
 	else if(!turn)
 		checks.emplace_back("no steering!");
+	else if(attributes.Get("required crew") > attributes.Get("bunks"))
```

Comment:
Shouldnt we do it the same way as the rest, by putting it in a variable? I imagine its done for readability

Reply:
Doesn't seem necessary. Many of those variables are used multiple times or in more complex checks. They're also all doubles. This is just a single greater than check. Even if I made a variable `bunks - crew`, this would still have to be `var < 0`.